### PR TITLE
Add a discussion of the difference between entering/exiting light transport

### PR DESCRIPTION
**Pull request by portsmouth**
Based on https://github.com/AcademySoftwareFoundation/OpenPBR/issues/110.

I added this discussion as a sub-section of the "Reduction to a mixture of lobes" section, as it is also quite technical and seems to make the most sense in the context of trying to do an implementation of the layering.

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1197,6 +1201,22 @@
 
 The full representation in code of this linear combination of BRDF/BTDF/BSSRDF lobes will depend on renderer-specific details.
 
+### Entering versus exiting
+
+As aspect that has been ignored in this approximation (in the non-thin-walled case) is the different light transport for rays incident from the top or the bottom side. In reality the physical effect of the layers differs in these cases. For entering rays:
```
Comment (virtualzavie) [inline index.html:1206]:
```suggestion
An aspect that has been ignored in this approximation (in the non-thin-walled case) is the different light transport for rays incident from the top or the bottom side. In reality the physical effect of the layers differs in these cases. For entering rays:
```

File: index.html [hunk 2]
```diff
@@ -1197,6 +1201,22 @@
...
+
+   - The fuzz reflection is un-tinted by the coat absorption.
```
Comment (virtualzavie) [inline index.html:1208]:
```suggestion
   - The fuzz reflection is not tinted by the coat absorption.
```
("un-tinted" sounds to me like a tint is removed somehow)

**Review by virtualzavie — APPROVED** [review]
Looks fair to me.
I've let two minor changes.

**Review by peterkutz — APPROVED** [review]
This sounds good to me. The guidelines are intuitive and reasonable, and the description seems to have the right level of detail.

**Review by jstone-lucasfilm — APPROVED** [review]

**Pull request merged**